akka-remote/src/main/resources/reference.conf

```diff
@@ -152,6 +152,18 @@ akka {
         # Level 1 strongly prefer low CPU consumption over low latency.
         # Level 10 strongly prefer low latency over low CPU consumption.
         idle-cpu-level = 5
+        
+        # Number of outbound lanes for each outbound association. A value greater than 1
+        # means that serialization can be performed in parallel for different destination
+        # actors. The selection of lane is based on consistent hashing of the recipient
+        # ActorRef to preserve message ordering per receiver.
+        outbound-lanes = 1
+        
+        # Total number of inbound lanes, shared among all inbound associations. A value 
+        # greater than 1 means that deserialization can be performed in parallel for 
+        # different destination actors. The selection of lane is based on consistent 
+        # hashing of the recipient ActorRef to preserve message ordering per receiver.
+        inbound-lanes = 1
```

Comment:
inbound is shared among all inbound associations, so it makes sense to have a larger value for inbound
outbound-lanes are per outbound association